### PR TITLE
Publishable version

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-  "main": "index.js",
+  "main": "dist/shell.js",
   "scripts": {
     "build": "tsc",
     "lint": "tslint -p .",
     "test": "ava -- dist/test/**.js"
   },
+  "files": [
+    "dist/",
+    "package.json"
```
Comment (vogre) [inline package.json]: I think `package.json` is going to be included regardless.

File: package.json [hunk 2]
```diff
@@ -1,13 +1,18 @@
 {
   "name": "specshell",
-  "version": "0.0.1",
+  "version": "0.0.1-rc.0",
   "description": "Control a program (typically a shell) over standard I/O for testing it",
-  "main": "index.js",
+  "main": "dist/shell.js",
   "scripts": {
     "build": "tsc",
```
Comment (vogre) [inline package.json]: Possibly a `rm -rf dist/ && tsc` with a separate target of `"build:watch": "tsc -w"`, alternatively as a prepublish target like in https://github.com/binaris/concord/blob/e3238f4590a15d838be75341b292b39c863aa2d0/package.json#L10.

File: package.json [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
     "lint": "tslint -p .",
     "test": "ava -- dist/test/**.js"
   },
+  "files": [
+    "dist/",
```
Comment (vogre) [inline package.json]: The issue with including `dist/` is that we include `dist/test`. As long as they are not that heavy it's not that bad, but we might try [project references](https://www.typescriptlang.org/docs/handbook/project-references.html) to have a separate `src/` and `test/`.

File: package.json (hunk 2 above)
Comment (arielshaqed) [reply package.json]: Did the latter (and added test-and-lint there, while I'm at it).

File: package.json (hunk 3 above)
Comment (arielshaqed) [reply package.json]: We do that elsewhere, too.  I'll give project references a try separately, opened #7.

File: package.json (hunk 1 above)
Comment (arielshaqed) [reply package.json]: You're right, but I'd have to look it up every time.

**Pull request merged**